Doc/library/ctypes.rst

```diff
@@ -1394,6 +1394,15 @@ the Windows error code which is managed by the :func:`GetLastError` and
 :func:`ctypes.set_last_error` are used to request and change the ctypes private
 copy of the windows error code.
 
+The *winmode* parameter is used on Windows to specify how the library is loaded
+(since *mode* is ignored). It takes any value that is valid for the Win32 API
+``LoadLibraryEx`` flags parameter. When omitted, the default is to use the flags
+that result in the most secure DLL load, avoiding issues such as DLL hijacking.
+
+.. versionchanged:: 3.8
+   Added *winmode* parameter.
+
```

Comment:
Wow. You went ahead and documented the new parameter and added tests. :+1: I thought you were going to leave it undocumented for now.

Reply:
The most common reason for adding a directory to `PATH`, calling `SetDllDirectoryW`, or changing the working directory has been loading a DLL with dependencies, which are often in the same directory. Maybe a note should be added that ctypes will now load dependencies from the same directory if `name` is passed as a qualified path.

Reply:
I tried to only half document it, so that we can change it in future without breaking our contract with users (part of the problem we're facing right now is that we never really defined DLL resolution as "undefined").

I'll add text like "passing the full path is recommended"